### ISSUE TITLE
Suppress host output

### ISSUE THREAD

**Issue by anishkny** [vot/dbclone#20]
Currently, calling `dbclone.export` displays the host output in STDOUT:

```
Exporting to /home/foo/bar
Host:        mongodb://<username>:<password>@XXXXX.mlab.com:25125/dbclone-test
Selected DB: dbclone-test
------------------
Starting export...
------------------
...
```

Offer an option to suppress this so we can run in CircleCI which currently [does not mask secrets in logs](https://discuss.circleci.com/t/masking-secrets-in-output-logs/22231/6). 

Alternately, lets move to TravisCI which does suppress secrets in logs.

**Comment by anishkny** [comment]
Yup that should cover it. There is [precedence](https://www.gnu.org/software/make/manual/html_node/Options-Summary.html) for flag names like `--silent`/`--quiet` or `--verbose` depending on the default we pick.

**Comment by vot** [comment]
Yeah I was thinking about these. We're verbose(-ish) by default here and I'm thinking about using `--silent` or `--quiet` to suppress all output.

I'd like to start with just a very explicit `--hide-secrets` and see how practical that is. It would apply only to two pieces of potentially sensitive data so it feels like a decent starting point.
My thinking is that this is version 0.0.3 right now so we still have enough flexibility to work out the best approach before reaching 1.0.0 :)

Feel free to disagree - just thinking out loud at this point.

**Comment by vot** [comment]
Hi @anishkny,

After going back and forth on this I sat down yesterday and started working on callback data not being passed correctly.
As I was moving the logic for printing to StdOut I cleaned up this as well.

Now when using programmatically this will not display the verbose information such as host and database name.
It's available through CLI behind a `--verbose` flag just as you suggested. Definitely feels far more intuitive than a non-standard approach I considered.

Anyway, these changes are now in master (PR #22). `example.js` has been updated to use values returned from the exposed methods. All of them have the same signature except for `drop` method (that's just a boolean).

Are you happy with closing this issue?
